graylog2-web-interface/src/views/components/sidebar/SidebarNavigation.jsx

```diff
@@ -12,16 +12,18 @@ type Props = {
   activeSection: ?SidebarSection,
   sections: Array<SidebarSection>,
   setActiveSectionKey: (sectionKey: string) => void,
+  sidebarIsPinned: boolean,
   toggleSidebar: () => void,
 };
 
-const Container: StyledComponent<{}, ThemeInterface, HTMLDivElement> = styled.div(({ theme }) => css`
-  background: ${theme.colors.gray[10]};
-  color: ${theme.colors.gray[80]};
-  box-shadow: 3px 0 3px rgba(0, 0, 0, 0.25);
+const Container: StyledComponent<{isOpen: boolean, sidebarIsPinned: boolean}, ThemeInterface, HTMLDivElement> = styled.div(({ isOpen, sidebarIsPinned, theme }) => css`
+  background: ${theme.colors.global.navigationBackground};
+  color: ${theme.utils.contrastingColor(theme.colors.global.navigationBackground, 'AA')};
+  box-shadow: ${(sidebarIsPinned && isOpen) ? 'none' : `3px 3px 3px ${theme.colors.global.navigationBoxShadow}`};
   width: 50px;
   height: 100%;
   position:relative;
+  z-index: 1031;
```

Comment:
What do you think about adding the `z-index` to the theme config as well?

Reply:
Yes, there is a plan to do that but just have not gotten to it yet